packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -12,17 +12,17 @@ import { SidePanel } from './SidePanel';
 import { GeneralTab } from './GeneralTab';
 import { InboundLineEdit } from './modals/InboundLineEdit/InboundLineEdit';
 import { getNextInboundStatus, isInboundEditable } from '../../utils';
-import { InboundShipmentItem } from '../../types';
+import { InvoiceLine, InboundShipmentItem } from '../../types';
 
 export enum ModalMode {
   Create,
   Update,
 }
 
-export const toItem = (summaryItem: InboundShipmentItem): Item => ({
-  name: summaryItem?.itemName,
-  code: summaryItem?.itemCode,
-  id: summaryItem?.itemId,
+export const toItem = (line: InboundShipmentItem | InvoiceLine): Item => ({
```

Comment:
Now that a `row` can be either an `InvoiceLine` or an `InboundShipmentItem`, have to change up the `onRowClick`